stubs/playsound/playsound.pyi

```diff
@@ -0,0 +1,7 @@
+from typing import Any
+
+logger: Any
+
+class PlaysoundException(Exception): ...
+
+playsound: Any
```

Comment:
While this is correct, it isn't very useful. It just says that there exists something named `playsound` in this library. It would be better to say that it is a function, and also describe the arguments it takes and what it returns. There's plenty of examples of doing this all over typeshed.

Because playsound does `str(path)` internally (https://github.com/TaylorSMarks/playsound/blob/9cf4af20caa5ae8586f88b65659681b24f0c4e69/playsound.py#L13), make sure that the first argument can also accept `pathlib.Path` objects. (There is also `_typeshed.StrPath`, but it isn't usable here because the code does `str(path)`, and `_typeshed.StrPath` doesn't guarantee that `str(path)` does anything meaningful.)